.gitignore

```diff
@@ -1 +1,5 @@
 node_modules
+# Intellij IDE project settings
+.idea
+# Because dogfooding
+.pr-train.yml
```

Comment:
Feel free to remove these changes, or undo them. I just needed to make them to have a clean working directory when rebasing and so on.